src/components/App/App.js

```diff
@@ -13,41 +14,68 @@ class App extends Component {
       movies: [],
       selectedMovie: {},
       isClicked: false,
-      error: ''
-    }
+      error: "",
+      hasError: false
+    };
   }
 
   componentDidMount = () => {
-    return allMoviesData().then(data => this.setState({ movies: data.movies }))
-    .catch(error => this.setState("ERROR"))
+    return allMoviesData()
+      .then((data) => this.setState({ movies: data.movies }))
+      .catch((error) => this.throwError("Oops! something went wrong. Please try again. If problem persists, send complaints to Robbie and Scott"));
```

Comment:
This is nice - I like that you have a function to handle how you're going to display the error to the user.